### ISSUE TITLE
Display expired transactions in activity with new "expired" state 

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#1899]
Related to https://github.com/hirosystems/explorer/issues/578 in the Explorer, we should show transactions that have simply not been picked up by the network in time (e.g. given congestion issues) and label them as "expired", instead of removing them from the activity view entirely.

**Comment by andresgalante** [comment]
There is an issue related to this one but I can't find it that suggest giving the user the ability to hide a transaction from their feed that is either duplicate or will never get confirmed

**Comment by markmhendrickson** [comment]
I think you mean this one https://github.com/hirosystems/stacks-wallet-web/issues/1867?

That functionality will serve a slightly different set of use cases, in that it targets transactions that are still pending vs. this one that targets ones no longer pending and marked as "dropped_stale_garbage_collect".

**Comment by aulneau** [comment]
I had both pending and dropped in mind when I made the issue, btw

**Comment by ginny-d** [comment]
Added Expired state/label here: https://www.figma.com/file/Li7qK8ZIG9c5dKSNOPO4iCtv/%E2%9D%96-Design-System?node-id=2582%3A3249

**Comment by markmhendrickson** [comment]
Do we not want to color this red similar to "Failed"?

![image](https://user-images.githubusercontent.com/28991/142213907-be29d0c7-2b8e-4b66-a68d-fe86a6f73827.png)

**Comment by ginny-d** [comment]
Where are you seeing this? Expired should be like this:

<img width="430" alt="Screenshot 2021-11-17 at 19 31 23" src="https://user-images.githubusercontent.com/78865845/142260914-c06b1a70-4702-47b0-a145-a45c34e8c776.png">

**Comment by markmhendrickson** [comment]
Right, I just mean that "Expired" is in grey here whereas "Failed" is in red. Might we want "Expired" in red as well since it indicates a sort of failure as far as the user is concerned? Albeit a particular one.

**Comment by ginny-d** [comment]
Updated figma with Expired label on red https://www.figma.com/file/Li7qK8ZIG9c5dKSNOPO4iCtv/%E2%9D%96-Design-System?node-id=2582%3A3249

**Comment by 314159265359879** [comment]
Lets also explain to users what it means? Something like this in the explorer and a questionmark they can click/hover over in the wallet?
![image](https://user-images.githubusercontent.com/33360391/220598020-6099bea4-d8a2-47af-a26c-5225960e59fd.png)

**Comment by 314159265359879** [comment]
![image](https://user-images.githubusercontent.com/33360391/220657881-8363e265-cf01-4b0e-9793-c8b4da6fb2dd.png)

**Comment by markmhendrickson** [comment]
cc @mica000
